fraud/proof.go

```diff
@@ -16,7 +16,7 @@ const (
 func (p ProofType) String() string {
 	switch p {
 	case BadEncoding:
-		return "BadEncoding"
```

Comment:
The "E" is still capitalized btw

Reply:
Oh, this comment landed on the deleted side of the diff. If you look the latest change there, you will see "badEncoding" with capitalized E in the middle.